### PR TITLE
10 user page can sort reviews

### LINKED ISSUE #10: User Story 17, User Show Page can sort reviews
```
As a Visitor,
When I visit a user's show page
I should also see links to sort reviews in the following ways:
- sort reviews newest first (descending chronological order)
- sort reviews oldest first (ascending chronological order)
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #12

child of #33

depends on #11

### PR DESCRIPTION
**Pull request by whois-jon-peterson**
Closes #10

### CODE REVIEW

File: app/models/review.rb [hunk 1]
```diff
@@ -16,4 +16,8 @@ def self.already_exists?(review, book)
     book.reviews.exists?(user: review.user)
   end
 
+  def self.user_by_date(user: nil, dir:)
```
Comment (jalena-penaligon) [inline app/models/review.rb:19]: Should dir: have a default value?
